spec/mailers/collections_mailer_spec.rb

```diff
@@ -9,6 +9,7 @@
   let(:work) { build_stubbed(:work, collection:) }
   let(:work_version) { build_stubbed(:work_version, work:) }
   let(:a_user) { build_stubbed(:user, name: 'Al Dente', first_name: 'Fred') }
+  let(:mailer_host) { Settings.host }
```

Comment:
Curious why we needed a new `let` value to inject the same string?

Reply:
I removed. I think I was doing something related and prefer lets in most cases, but it doesn't buy us anything here.